dags/main_summary.py

```diff
@@ -251,6 +251,21 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/update_glue.sh",
     dag=dag)
 
+taar_dynamo = EMRSparkOperator(
+    task_id="taar_dynamo",
+    job_name="TAAR DynamoDB loader",
+    execution_timeout=timedelta(hours=14),
+    instance_count=6,
+    owner="vng@mozilla.com",
+    email=["mlopatka@mozilla.com", "vng@mozilla.com", "sbird@mozilla.com"],
+    env=mozetl_envvar("taar_dynamo", {
+        "date": "{{ yesterday_ds_nodash }}"
```

Comment:
if you use `yesterday_ds_nodash`, this will always run for yesterday's date, regardless of what the scheduled execution date is for the current task run (e.g. if we find an error in a main_summary run a week later and re-run, this task will still run for yesterday's date, even though the DAG is running for last week). You probably want `ds_nodash` instead.

Reply:
Thanks - I've replaced this now.